okta/user.go

```diff
@@ -258,11 +258,11 @@ func listUserOnlyRoles(ctx context.Context, c *okta.Client, userID string) (user
 		return
 	}
 	for _, role := range roles {
-		if role.AssignmentType == userScope && role.Type != "CUSTOM" {
+		if role.Type != "CUSTOM" {
 			userOnlyRoles = append(userOnlyRoles, role)
 		}
 	}
-	return
+	return userOnlyRoles, resp, err
```

Comment:
The method ` listUserOnlyRoles` has named return types so `return userOnlyRoles, resp, err` is redundant and should only be `return` (personally, I'm not a fan of named return types given it makes bare return states look odd/illogical)

Reply:
i've never seen that functionality before, thanks for the info 😄